perun-core/src/test/java/cz/metacentrum/perun/core/impl/DefaultBlockedLoginCheckerTest.java

```diff
@@ -0,0 +1,74 @@
+package cz.metacentrum.perun.core.impl;
+
+import cz.metacentrum.perun.core.AbstractPerunIntegrationTest;
+import cz.metacentrum.perun.core.api.Attribute;
+import cz.metacentrum.perun.core.api.BeansUtils;
+import cz.metacentrum.perun.core.api.CoreConfig;
+import cz.metacentrum.perun.core.api.User;
+import cz.metacentrum.perun.core.api.exceptions.AttributeDefinitionExistsException;
+import cz.metacentrum.perun.core.api.exceptions.InternalErrorException;
+import cz.metacentrum.perun.core.api.exceptions.WrongAttributeAssignmentException;
+import cz.metacentrum.perun.core.api.exceptions.WrongAttributeValueException;
+import cz.metacentrum.perun.core.api.exceptions.WrongReferenceAttributeValueException;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThrows;
+
+public class DefaultBlockedLoginCheckerTest extends AbstractPerunIntegrationTest {
+	private final static String CLASS_NAME = "DefaultBlockedLoginChecker.";
+
+	private User user;
+	private Attribute attr;
+
+	DefaultBlockedLoginChecker defaultBlockedLoginChecker;
+
+	@Before
+	public void setUp() throws Exception {
+		defaultBlockedLoginChecker = new DefaultBlockedLoginChecker(perun);
+
+		String login = "testLogin";
+
+		setUser();
+		setLoginNamespaceAttribute(login);
+		addDefaultBlockedLoginToCoreConfig(login);
+	}
+
+	@Test
```

Comment:
Maybe add another test for a case when the exception is not thrown? If not, it feels unnecessary to have private setup methods.